src/WebJobs.Extensions.CosmosDB/CosmosDBAttribute.cs

```diff
@@ -108,6 +108,16 @@ public CosmosDBAttribute(string databaseName, string collectionName)
         /// </summary>
         public bool UseMultipleWriteLocations { get; set; }
 
+        /// <summary>
+        /// Optional.
+        /// Enables the use of JsonConvert.DefaultSettings in the monitored Azure Cosmos DB collection.
+        /// <remarks>
+        /// This setting only applies to the monitored collection and the consumer to setup the serialization used in the monitored collection.
+        /// The JsonConvert.DefaultSettings must be set in a class derived from CosmosDBWebJobsStartup.
```

Comment:
Could you add a <example> of how to achieve it?

Reply:
done